lib/ansible/modules/windows/win_domain.py

```diff
@@ -71,6 +72,26 @@
     type: str
     choices: [ Win2003, Win2008, Win2008R2, Win2012, Win2012R2, WinThreshold ]
     version_added: '2.8'
+  parent_domain_name:
+    description:
+    - Specifies the parent domain while creating a new domain in a forest.
+    - Has to be the parent domain's fqdn
+    - It can't be used with forest mode as we don't create a new forest.
```

Comment:
Please look [here](https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#linking-within-module-documentation) to make the reference for the parameter of `forest_mode`